### PR TITLE
Panelapp links

### LINKED ISSUE #5036: PanelApp links on gene and panel pages
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**
We already create panel app links for genes on variants, but it would not hurt to have these also on the gene page. This would be part A, and should be really straightforward. For part B it would also be cool to link back to the PanelApp panel entities, either from panel name or from some saved artifact to the downloaded panel.

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Closes #5036

### On panel page
image

### On gene page

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/genes/templates/genes/gene.html [hunk 1]
```diff
@@ -84,6 +84,12 @@ <h1>
                       <a target="_blank" href={{ record.entrez_link }} referrerpolicy="no-referrer" rel="noopener" target="_blank"> {{ entrez_id }}</a>
                     </span>
                   </li>
+                  <li class="list-group-item">
+                    PanelApp
+                    <span class="float-end">
+                      <a target="_blank" href={{ record.panelapp_link }} referrerpolicy="no-referrer" rel="noopener" target="_blank"> {{ symbol }} </a>
```
Comment (dnil) [inline scout/server/blueprints/genes/templates/genes/gene.html:90]: 💯

File: scout/parse/panelapp.py [hunk 2]
```diff
@@ -90,6 +91,7 @@ def parse_panelapp_panel(
     gene_panel["display_name"] = " - ".join([panel_info["name"], f"[{confidence.upper()}]"])
     gene_panel["institute"] = institute
     gene_panel["panel_type"] = ("clinical",)
+    gene_panel["description"] = f"{PANELAPP_PANELS_URL}{panel_id}"
```
Comment (dnil) [inline scout/parse/panelapp.py:94]: 😂 Sure, better than before! I had imagined a clickable one. 😜

**Review by dnil — APPROVED** [review]
Sure, sure. I have not considered how much work actually adding the link to the panel page as well would be, but I trust it looked complicated.

File: scout/parse/panelapp.py (hunk 2 above)
Comment (northwestwitch) [reply scout/parse/panelapp.py:94]: Me too, but it's more complicated than it seems..

**Pull request merged**